js/bittrex.js

```diff
@@ -307,6 +326,7 @@ module.exports = class bittrex extends Exchange {
                         'max': undefined,
                     },
                     'leverage': {
+                        'min': 1,
                         'max': 1,
                     },
```

Comment:
I think, if they don't have `leverage`, then we should either remove `leverage` entirely, or set it to `undefined`, or set `min` and `max` to `undefined`.

Reply:
https://github.com/ccxt/ccxt/pull/11302#discussion_r784654212